packages/cashier/src/pages/p2p-cashier/p2p-cashier.jsx

```diff
@@ -42,7 +45,16 @@ const P2PCashier = ({
 
     React.useEffect(() => {
         const url_params = new URLSearchParams(location.search);
-        const passed_order_id = url_params.get('order');
+        let passed_order_id;
+
+        setActionParam(url_params.get('action'));
+        setCodeParam(url_params.get('code'));
+
+        if (url_params.has('order_id')) {
+            passed_order_id = url_params.get('order_id');
+        } else if (url_params.has('order')) {
+            passed_order_id = url_params.get('order');
```

Comment:
What is the difference between `order` and `order_id`? 🤔

Reply:
Different email links give us different verification_code, for this one it's order_id and it's what works for mobile.